content/shell/android/shell_apk/src/org/chromium/content_shell_apk/ContentShellActivity.java

```diff
@@ -73,7 +73,9 @@ protected void onCreate(final Bundle savedInstanceState) {
         mWindowAndroid.setAnimationPlaceholderView(
                 mShellManager.getContentViewRenderView().getSurfaceView());
 
```

Comment:
Is there a clear `if(cobalt)` conditional we can use in Java code to do conditionals ? Do gn flags get propagated into Java somehow at all ?

Reply:
Java doesn't do macros and usually uses either dependency injection or static flags(like the checks for API version) to do this. Cobalt never really needed `if(cobalt)` in java because we wrote all our java code.

The code here should be reverted not long after, but we indeed have a small question of if we want to gate changes to Chromium java code with some runtime flag or just minimize the changes.